src/test/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/cost/PatternCostCalculatorTest.java

```diff
@@ -92,42 +109,36 @@ public void testPreferencesPenaltyForRoute(String testCaseDescription) {
     RoutePenaltyTC tc = new RoutePenaltyTC(testCaseDescription);
 
     var schedule = tc.createSchedule();
-    var routeCostCalculator = tc.createRouteCostCalculator();
+    var costCalculator = tc.createCostCalculator();
 
     int defaultArrCost = transitArrivalCost(schedule, defaultCostCalculator);
     int defaultBoardCost = boardingCost(schedule, defaultCostCalculator);
-    int arrCost = transitArrivalCost(schedule, routeCostCalculator);
-    int boardCost = boardingCost(schedule, routeCostCalculator);
-
-    if (tc.unPrefRoute) {
-      var expectedArr = unprefCostFn.apply(TRANSIT_TIME);
-      var errorMessageArr = String.format("Invalid arrival cost: %s", tc);
-      assertEquals(expectedArr, arrCost - defaultArrCost, errorMessageArr);
-      /*
-      TODO: separate boarding cost calculation can be tested when boarding and arrival cost
-            constants are differentiated
-      var expectedBoard = unprefCostFn.apply(0);
-      var errorMessageBoard = String.format("Invalid boarding cost: %s", tc);
-      assertEquals(expectedBoard, boardCost - defaultBoardCost, errorMessageBoard);
-      */
+    int arrCost = transitArrivalCost(schedule, costCalculator);
```

Comment:
This is not something that is your fault but I was quite confused how this test is supposed to work. I think we want to have a chat in the dev meeting about it.